src/_imagingft.c

```diff
@@ -630,7 +630,7 @@ font_getsize(FontObject* self, PyObject* args)
 
 
     for (x = i = 0; i < count; i++) {
-        int index, error;
+        int index, error, offset, x_advanced;
```

Comment:
What's the reason for moving these declarations before use?

Reply:
`x_advanced` is new, not moved. I'm happy to move the declarations down if you like, I'm just going with the convention established in the code - I moved the `offset` declaration up because it is no longer inside a conditional block.